### PR TITLE
Allow admins to add/remove users from other realms

### LINKED ISSUE #1003: Delete-from-realm link in system-admin user page
### Summary
We recently added a user-details page to system admin which shows a list of realms. Let's add in a link to 'remove from realm' similar to the my-account page.

leaverealm

Currently we have:
`/admin/realms/{{$realm.ID}}/leave`
and
`/users/{{.ID}} data-method="DELETE"`

One assumes currentUser and specifies realmId, the other assumes currentRealm and specifies userId.
It would be nice to not assume anything for the /admin path.

We'd want to add the link here: https://github.com/google/exposure-notifications-verification-server/blob/86df37ac5548473b62ca0be36663242812572d90/cmd/server/assets/admin/users/show.html#L55

### PR DESCRIPTION
**Pull request by sethvargo**
Fixes #1003

**Release Note**

```release-note
Allow admins to add/remove users from other realms
```

### CODE REVIEW

File: cmd/server/assets/admin/users/show.html [hunk 1]
```diff
@@ -47,24 +47,30 @@ <h6 class="mb-2">System admin</h6>
       </div>
     </div>
 
-    <div class="card mb-3 shadow-sm">
-      <div class="card">
+    {{if $user.Realms}}
```
Comment (whaught) [inline cmd/server/assets/admin/users/show.html:50]: Alternatively leave the card and
```
{{else}
This user is not a member of any realms
```

might make the UI more consistent / clear

**Review by whaught — COMMENTED** [review]
/lgtm
/hold depending on how you feel about my other comment

File: cmd/server/assets/admin/users/show.html (hunk 1 above)
Comment (sethvargo) [reply cmd/server/assets/admin/users/show.html]: Done

**Review by whaught — APPROVED** [review]

**Pull request merged**